diff --git a/node_modules/expo-image-picker/android/build.gradle b/node_modules/expo-image-picker/android/build.gradle
index 1fe33a1..badc8a4 100644
--- a/node_modules/expo-image-picker/android/build.gradle
+++ b/node_modules/expo-image-picker/android/build.gradle
@@ -87,7 +87,7 @@ dependencies {
   implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:${getKotlinVersion()}"
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.3")
   implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.1")
-  implementation "com.github.CanHub:Android-Image-Cropper:1.1.1"
+  implementation "com.github.CanHub:android-image-cropper:e81a7d8eb1d"
   implementation "androidx.exifinterface:exifinterface:1.3.3"
   api "androidx.legacy:legacy-support-v4:1.0.0"
   api 'commons-codec:commons-codec:1.10'
